cmd/client/main.go

```diff
@@ -135,7 +139,14 @@ func (o *GrpcProxyClientOptions) Validate() error {
 		return fmt.Errorf("please do not try to use ephemeral port %d for the proxy server port", o.proxyPort)
 	}
 	if o.proxyPort < 1024 {
-		return fmt.Errorf("please do not try to use reserved port %d for the proxy server port", o.proxyPort)
+		if o.proxyUdsName == "" {
+			return fmt.Errorf("please do not try to use reserved port %d for the proxy server port", o.proxyPort)
+		}
+	}
+	if o.proxyUdsName != "" {
```

Comment:
nit: move the two if's to one line to reduce indention

Reply:
Add a check to verify that clientCert, CA, key are not set.

Reply:
Not merging if's as I am following your second suggestion :D